comps/cores/proto/docarray.py

```diff
@@ -110,14 +102,8 @@ class Config:
         json_encoders = {np.ndarray: lambda x: x.tolist()}
 
 
-class SearchedMultimodalDoc(BaseDoc):
-    retrieved_docs: List[TextImageDoc]
-    initial_query: str
-    top_n: int = 1
-    metadata: Optional[List[Dict]] = None
-
-    class Config:
-        json_encoders = {np.ndarray: lambda x: x.tolist()}
+class SearchedMultimodalDoc(SearchedDoc):
+    metadata: List[Dict[str, Any]]
```

Comment:
@BaoHuiling Should we have `metadata: List[Dict[str, Any]] = None`?

Reply:
That will make this field optional, then why user not use SearchedDoc directly?

Reply:
Agree on optional. However, using SearchedDoc directly without metadata variable cannot enforce future development to make use of the variable `metadata`. One can simply extends SearchedDoc with additional image, audio, etc... for the purpose of multimodality, making this difficult to inherit and reuse in many other downstream microservice.   
It is ok to leave it without `None`. I think the results will be the same.

Reply:
Agree. If we need it in the future, issue another PR is not complicated.